src/layout/questionnaire-list/components/questionnaire-list.jsx

```diff
@@ -88,6 +119,28 @@ const QuestionnaireList = props => {
           </div>
         )}
       </div>
+      <ReactModal
+        ariaHideApp={false}
+        shouldCloseOnOverlayClick={false}
+        isOpen={showPopup}
+        onRequestClose={handleClosePopup}
+        contentLabel="Alert Save"
+        className="popup-duplication"
+      >
+        <div className="popup-header">
+          <h3>{Dictionary.dupliquate}</h3>
+          <button type="button" onClick={handleClosePopup}>
+            <span>X</span>
+          </button>
+        </div>
+        <div className="popup-body">{Dictionary.duplicateQuestion}</div>
```

Comment:
You maybe have to insert the name of the questionnaire in the message. Because you have ""titre à insérer ici"" in the string defined in the i18n configuration object.